x-pack/plugins/apm/server/lib/environments/get_all_environments.ts

```diff
@@ -27,8 +27,8 @@ export async function getAllEnvironments({
   includeMissing?: boolean;
 }) {
   const spanName = serviceName
-    ? 'get_all_environments_for_all_services'
-    : 'get_all_environments_for_service';
+    ? 'get_all_environments_for_service'
+    : 'get_all_environments_for_all_services';
```

Comment:
Was this a bug previously?

Reply:
It's the span name, I want to have different span names depending if it's trying to get envs for one service or all services (as the performance characteristics are different). But I accidentally switched them around. So not a user-facing bug, but an instrumentation error. The only consequence is that we can't compare these specific span operations from before and after this PR.